src/components/number-input/number-input.js

```diff
@@ -12,18 +12,16 @@ const NumberInput = ({ onChangeQuantity, quantity }) => {
     <div className={styles.root} data-cy='cart-item-quantity'>
       <Button
         className={styles.button}
-        onClick={() => onChangeQuantity(-1)}
+        onClick={() => onChangeQuantity(quantity - 1)}
```

Comment:
quantity is zero by default
<img width="180" alt="Screenshot 2021-02-12 at 15 40 17" src="https://user-images.githubusercontent.com/68334137/107775292-c2e82a00-6d48-11eb-81ef-b80935d0fe52.png">